.github/workflows/release-drafter.yml

```diff
@@ -0,0 +1,18 @@
+name: Release Drafter
+
+on:
+  push:
+    # branches to consider in the event; optional, defaults to all
+    branches:
+      - master
+
+jobs:
+  update_release_draft:
+    runs-on: ubuntu-latest
+    steps:
+      # Drafts your next Release notes as Pull Requests are merged
+      # https://github.com/raboof/release-drafter/releases
+      # fork of https://github.com/release-drafter/release-drafter/releases
+      - uses: raboof/release-drafter@v5
```

Comment:
I don't get it, why a fork?  Is it a security thing?  Or is it for behaviour changes?

Reply:
No behavior changes, security indeed - I just didn't feel quite comfortable giving away the keys to the kingdom to whoever gains access to that repo. Though of course the same could be said about my fork...